src/Parser/VengeanceDemonHunter/CombatLogParser.js

```diff
@@ -122,11 +131,20 @@ class CombatLogParser extends MainCombatLogParser {
       });
     }
 
+    if (spiritBombUptimePercentage < 1.0 && this.selectedCombatant.hasTalent(SPELLS.SPIRIT_BOMB_TALENT.id)) {
+      results.addIssue({
```

Comment:
Please use the new issue syntax, when moving this suggestion to its own module this will also be the only way, see the Holy Paladin spec for examples.